bin/taskcluster/task_decision.js

```diff
@@ -89,7 +92,7 @@ function createTask(name, yml) {
 
     routes: [
       "index.project.devtools.branches." + process.env.GITHUB_HEAD_REPO_BRANCH + "." + name,
-      "index.project.devtools.branches." + process.env.GITHUB_HEAD_REPO_SHA + "." + name
+      "index.project.devtools.revisions." + process.env.GITHUB_HEAD_REPO_SHA + "." + name
```

Comment:
This was wrong. BTW, you can see artifacts / debug routes on this page:
https://tools.taskcluster.net/index/artifacts/#project.devtools/project.devtools